### DISCUSSION TITLE
Get page counters from USB printers [Q&A]

### DISCUSSION THREAD

**Discussion by mmarodin** [glpi-project/glpi-agent#668] [Q&A] [answered]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Hi, is it possible to retrieve page counter from USB printers connected to the device where glpi-agent is running?
I've some HP laserjet printers connected via USB.

Please let me know, thanks.
Bye, Morgan

**Comment by g-bougard** [comment, marked as answer]
Hi @mmarodin 

no this is not possible via USB and even the inventory format doesn't support this data.

The only way to obtain these counters is via networkinventory task. So if the printer is also connected via network I guess you should be able to get them this way.
